specification/basic_concepts/basic_concepts.tex

```diff
@@ -1,79 +1,209 @@
+    runs the risk of encountering port identifier conflicts that cannot be resolved without resorting to help
+    of such external party, since the designers of the system do not have control over their fixed port identifiers.
+    This is why the specification strongly discourages the use of fixed unregulated private port identifiers.
+    If a data type definition is ever disclosed to any other party (i.e., a party that did not author it)
+    or to the public at large, such data type should be devoid of a fixed port identifier.
+}.
+This restriction shall be followed at all times by all compliant implementations and
+systems\footnote{
+    In general, private unregulated fixed port identifiers are collision-prone by their nature, so they should
+    be avoided unless there are very strong reasons for their usage and the authors fully understand the risks.
+}.
+
+\begin{UAVCANSimpleTable}{Data type taxonomy}{|l|X|X|}
+    & Regulated & Unregulated \\
+    \bfseries{Public}
+    &
+    Standard and contributed (e.g., vendor-specific) definitions.\newline
+    Fixed port identifiers are allowed; they are called \emph{``regulated port ID''}.
+    &
+    Definitions distributed separately from the UAVCAN specification.\newline
+    Fixed port identifiers are \emph{not allowed}.
+    \\
+
+    \bfseries{Private}
+    &
+    Nonexistent category.
+    &
+    Definitions that are not available to anyone except their authors.\newline
+    Fixed port identifiers are permitted (although not recommended) as long as the definition is only used by
+    its authors; they are called \emph{``unregulated fixed port ID''}.
+    \\
+\end{UAVCANSimpleTable}
+
+DSDL processing tools shall prohibit unregulated fixed port identifiers by default,
+unless they are explicitly configured otherwise.
+
+Each of the two sets of valid port identifiers (which are subject identifiers and port identifiers) are
+segregated into three categories: application-specific (i.e., non-fixed or unregulated) identifiers,
+regulated non-standard port identifiers, and standard port identifiers.
+Fixed unregulated port identifiers can be allocated in the same range where application-specific identifiers are.
+The ranges are documented in chapter \ref{sec:application_layer}.
+
+Data type authors that desire to release regulated data type definitions or contribute to the standard data
+type set should contact the UAVCAN maintainers for coordination.
+The maintainers will choose unoccupied fixed port identifiers for use with the new definitions, if necessary.
+Since the set of regulated definitions is maintained in a highly centralized manner,
+it can be statically ensured that no identifier collisions will take place within it;
+also, since the identifier ranges used with regulated definitions are segregated from the rest,
+regulated port ID will not conflict with any other compliant UAVCAN node or system\footnote{
```

Comment:
```suggestion
regulated port-IDs will not conflict with any other compliant UAVCAN node or system\footnote{
```
In general I'd use a hyphen when discussing types of port identifiers to be explicit about what term modifies which (e.g. is the port or the identifier regulated: "regulated-port identifier" or "regulated port-identifier")